backends/server/routes/userRoutes.ts

```diff
@@ -0,0 +1,73 @@
+import express, { Router, Request, Response } from "express";
+import { PrismaClient } from "@prisma/client";
+
+const router = Router();
+const prisma = new PrismaClient();
+
+// Create a new user
+router.post("/", async (req: Request, res: Response) => {
+  try {
+    const { firstName, lastName, email, password, address, phoneNumber } =
+      req.body;
+    const newUser = await prisma.user.create({
+      data: { firstName, lastName, email, password, address, phoneNumber, createdAt: new Date(), updatedAt: new Date() },
```

Comment:
on the prisma.schema file you have this line:
`createdAt DateTime @default(now())`

```suggestion
      data: { firstName, lastName, email, password, address, phoneNumber, updatedAt: new Date() },
```

So you can remove `createdAt` here